### PR TITLE
SDT-479 SDK Housekeeping

### PR DESCRIPTION
**Pull request by flxfischer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,101 @@
...
+
+## Usage 🤩
+Import the framework in your project:
+```swift
+import MoveSingleAPISwift
+```
+
+[Create an MoveSingleAPI key](https://move.ai) and add it to your configuration:
+```swift
+let move = Move(apiKey: "TOKEN")`
```
Comment (sjweekes) [inline README.md:16]: remove the trailing `

File: README.md [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
+    videoURL: // URL to local video file,
+    enhancementData: enhancementData
+)
+```
+
+## Licence 📥
+
+The MIT License (MIT)
+
+Copyright (c) 2023 Move
```
Comment (sjweekes) [inline README.md:95]: Change Move to Move AI

**Review by sjweekes — CHANGES_REQUESTED** [review]
Some small changes

File: README.md (hunk 1 above)
Comment (sjweekes) [reply README.md:16]: Also change 'Create an MoveSingle..." with "Create a MoveSingle..."

File: README.md [hunk 3]
```diff
@@ -1,3 +1,101 @@
...
+```swift
+import MoveSingleAPISwift
+```
+
+[Create an MoveSingleAPI key](https://move.ai) and add it to your configuration:
+```swift
+let move = Move(apiKey: "TOKEN")`
+```
+
+### Create simple Take
```
Comment (sjweekes) [inline README.md:19]: Can we add a line explaining what a take is, using the copy from the Python SDK : 
"Takes are a way to define a recording session. A single take can be associated with a video file and optionally additional files."

File: README.md [hunk 4]
```diff
@@ -1,3 +1,101 @@
...
+```
+
+### Upload Take
+Uploads the video file and the meta data and creates a new take in the system.
+```swift
+try await take.upload()
+```
+
+### Create Job
+Take has to be uploaded. This creates a new job that runs the video through the pipeline to generate the animation.
```
Comment (sjweekes) [inline README.md:34]: Can we add a line explaining what a job is, using the copy from the Python SDK :
"Jobs are the processing entity in MoveUGC. By creating a Job, you can initiate the processing for a take. A job can have multiple output types, currently the only output types supported are mp4, usdc and fbx files."

**Review by sjweekes — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,3 +1,101 @@
...
+    try await take.currentJob?.outputFiles[.preview]?.download()
+    let localPreviewURL = await take.currentJob?.outputFiles[.preview]?.localURL
+}
+```
+
+### Create Take with EnhancementData
+Enhancement Data is an optional Array of `EnhancementData` for every frame of the video.
+
+#### CameraDesignData
+`CameraDesignData` holds the internal size and position data of the camera. The information is extracted a given `CMSampleBuffer`.
```
Comment (sjweekes) [inline README.md:53]: can we change The information is extracted a given... 
to 
The information is extracted from a given...

File: README.md (hunk 3 above)
Comment (flxfischer) [reply README.md:19]: I change it to "additional enhancement data" to make it more clear.

**Review by sjweekes — APPROVED** [review]

**Pull request merged**